project2/styles.css

```diff
@@ -0,0 +1,121 @@
+* {
+    box-sizing: border-box;
+    transition: all 1s;
+}
+
+body {
+    background-color: #111111;
+    color: e0e0e0;
+    font-family: Rubik, Sans-serif;
+    font-weight: 400;
+}
+
+.info {
+    background-color: #2b2b2b;
+    color: #e0e0e0;
+    max-width: 500px;
+    max-height: 350px;
+    padding: 15px;
+    position: relative;
+    top: 30px;
+    left: 425px;
```

Comment:
So, here, I've added `background-color: red;` to the `div` on line 14 of your HTML:
![image](https://user-images.githubusercontent.com/2653980/67912996-15a4cf00-fb62-11e9-9ad3-1529aa4dcd3a.png)

It appears you've relatively positioned everything relative to the body in order to achieve your centered layout. The better way to do this is to add a class to your `div` and then give that `margin-right: auto; margin-left: auto; width: 500px;` (which is the width of your image) -- then you could do _a lot_ less item-by-item positioning of your items. It's hard to explain this in a line-by-line PR review, so I took your markup and styles, made some changes and notes to the CSS in [this CodePen](https://codepen.io/angeliquejw/pen/0c01cddece84edb79165e8f4c556d286?editors=0100) -- once you've had a chance to review and compare, happy to discuss and give more info 👍